inc/views/post_layout.php

```diff
@@ -207,16 +214,35 @@ public function render_cover_header() {
 		if ( $title_mode ) {
 			$title_meta_wrap_classes[] = 'nv-is-boxed';
 		}
-		$alignment = apply_filters( 'neve_post_title_alignment', '' );
-		if ( $container_mode === 'contained' ) {
-			$container_class[] = $alignment;
-		} else {
-			$main_class[] = $alignment;
+
+		/**
+		 * Filters the post title styles to override specific styles.
+		 *
+		 * @param string $style The styles for the title.
+		 * @param string $context The context of the layout (e.g. 'cover', 'normal'). Default is 'normal'.
+		 *
+		 * @since 3.1.0
+		 */
+		$cover_style = apply_filters( 'neve_post_title_alignment_style', $cover_style, 'cover' );
+		if ( ! empty( $cover_style ) ) {
+			$cover_style = 'style="' . $cover_style . '"';
+		}
+
+		/**
+		 * Keep this for legacy purpose
+		 */
+		if ( ! neve_is_new_skin() ) {
+			$alignment = apply_filters( 'neve_post_title_alignment', '' );
+			if ( $container_mode === 'contained' ) {
+				$container_class[] = $alignment;
+			} else {
+				$main_class[] = $alignment;
+			}
 		}
 
 		$meta_before = get_theme_mod( 'neve_post_cover_meta_before_title', false );
 
-		echo '<div class="' . esc_attr( implode( ' ', $main_class ) ) . '" ' . $cover_style . '>'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		echo '<div class="' . esc_attr( implode( ' ', $main_class ) ) . '" ' . wp_kses_post( $cover_style ) . '>';
```

Comment:
Now that we don't use the alignment classes, we can remove the `$main_class` variable and replace it with `nv-post-cover`